user_sync/connector/directory_okta.py

```diff
@@ -112,7 +112,7 @@ def load_users_and_groups(self, groups, extended_attributes):
         is_using_source_filter = False
         source_filters = options['source_filters']
         source_filter = source_filters.get('all_users_filter')
-        if (source_filter != None):
+        if source_filter is not None:
```

Comment:
truly idiomatic Python would be:
```
if source_filter:
```
unless you specifically want to include the case where source_filter is an empty list or empty string (which I'm pretty sure you don't).